<html>
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="style.css">
  </head>
  
  <div class="hplc">
    <div class="caption">
      <span class="border">Sina Marquardt, M.Sc. Biology</span>
    </div>
    <div class="caption_subtitle">
      <span class="border_subtitle">Biochemistry. Molecular Biology. Bioinformatics.</span>
    </div>
  </div>
  
  <div class="textSection">
    <h3><a id="Jumpup" style="text-decoration: none>Professional Experience</a></h3>
    
    <table style="width:850px">
      <tr>
        <td>
          <h4><a href="#Jump1">Scientific employee</a></h4>
          <p style="line-height: 0%">IPK Gatersleben</p>
          <p style="font-size: 14px">Juni 2018 - November 2020</p>
        </td>
      </tr>
      
      <tr>
        <td>
          <h4><a href="#Jump2">Master thesis</a></h4>
          <p style="line-height: 0%">Fraunhofer IME</p>
          <p style="font-size: 14px">April 2017 - Dezember 2017</p>
        </td>
      </tr>
                 
      <tr>
        <td>
          <h4><a href="#Jump3">Master of Science, Biology</a></h4>
          <p style="line-height: 0%">RWTH Aachen University</p>
          <p style="font-size: 14px">October 2012 - January 2018</p>
        </td>
      </tr>
      
      <tr>
        <td>
          <h4><a href="#Jump4">Research Internship</a></h4>
          <p style="line-height: 0%">RWTH University Aachen</p>
          <p style="font-size: 14px">Juli 2016 - September 2016</p>
        </td>
      </tr>

      
      <tr>
        <td>
          <h4>Bachelor thesis</h4>
          <p style="line-height: 0%">RWTH Aachen University Hospital</p>
          <p style="font-size: 14px">April 2015 - September 2015</p>
        </td>
      </tr>

      <tr>
        <td>
          <h4>Bachelor of Science, Biology</h4>
          <p style="line-height: 0%">RWTH Aachen University</p>
          <p style="font-size: 14px">October 2012 - September 2015</p>
        </td>
      </tr>
    
    </table>
    
    <p></p><br>
    <h3>Technical Skills</h3>
    <table style="width:850px">
      <tr>
        <td>
            <h5>Biological Methods:</h5>
              <p>- Cultivation and Transformation of microorganisms and plants</p>
              <p>- Evaluation of spectrometric data</p>
              <p>- Fluoreszenz- and Massspectrometry (LC-MS, MALDI-TOF-MS)</p>
              <p>- Metabolite profiling with UPLC-MS/MS</p>
              <p>- Molecular biologicl methods (PCR, Genome Walking, Cloning, Preparation of genomic and plasmid-DNA, DNA and Protein Gel Electrophoresis, Immunoblotting)</p>
              <p>- Purification of semipolar metabolites from plant material</p>              
               
        </td>
      </tr>
      
      <tr>
        <td>      
            <h5>Software and Programming Skills:</h5>
              <p>- Sequence Aanalysis (CloneManager)</p>
              <p>- Statistical Data Analysis (R, SPSS)</p>
              <p>- Laboratory Information Management System (LIMSOPHY)</p>
              <p>- Microsoft Office (Word, Excel, PowerPoint)</p>
              <p>- Mass Spectrometric Analysis (Bruker Software: DataAnalysis, Metaboscape, QuantAnalysis)</p>
              <p>- Graphical Editing (inkscape, GIMP)</p>
              <p>- Programming (Python and HTML basics)</p>
        </td>
      </tr>
    </table>

    <h3><p></p><br><p>Contact</p></h3>
    
    <p>
      <a href="http://linkedin.com/in/sina-marquardt-8b4225156" style="text-decoration: underline" class="author-social" target="_blank">linkedin</a>
      <a href="https://github.com/sina-marquardt" style="text-decoration: underline" target="_blank">github</a>
    </p>
  </div>
  
  <div class="hplc_room">
  </div>
  
  <div class="textSection">
    <h3><a id="Jump1" style="text-decoration: none">Scientific employee</a></h3>
    <h5>at the Leibniz Institute for Plant Genetics and Crop Plant Research IPK Gatersleben</h5>
    <h5>Department of Physiology and Cell Biology / Applied Biochemistry</h5>
    <h5><p>Project: BARLEY BIODIVERSITY - Increasing biodiversity and productivity of barley cultivars using a sensor-based high-throughput phenotyping system in the field and association mapping with two wild barley introgression libraries</p></h5>
      <p style="font-size: 14px">June 2018 - March 2020</p>
      <p>The project was part of the initiative „Innovative Pflanzenzüchtung im Anbausystem“ (IPAS)</p>
      <p>Tasks and methods: My task in the project was the evaluation and statistical analysis of multi-modal data from a large barley maping population.The data have been obtained</p><p>using various analytical techniques and across several years. In terms of analytical complexity, the project included handling large and diverse data structures, including:</p><p>genotype data, mass spectrometry profiles, phenotype descriptors and quantitative phenotypes, as well as analysis of the data variation and reproducibility across different</p><p>cultivation years and plant developmental stages.</p> 
      <p></p><br>
      <p>Analysis of metabolic data: Identification of metabolic features in the mass spectrometry data obtained by HPLC-MS/MS, providing a comprehensive profile of semi-polar</p><p>compounds accumulated in barley leaves</p>
      <p>Data normalization:Data normalization using multiple internal standards by fitting a multivariate linear model to a set of reference</p><p>internal standards and removing the fitted variance from the target variables.</p>
      <p>Quality control: Quality control (QC) samples were used to estimate the data reproducibility and technical error across analytical batches and cultivation seasons.</p> 
      <p>ANOVA: Analysis of variance has been performed to highlight effects of different experimental factors: e.g. genotype, batch, developmental stage.</p>  
      <p>Pearson correlation analysis: Correlation analysis has been performed between measured variables using Pearson’s correlation coefficient. Homogeneity of the correlations</p><p>across cultivation seasons has been analysed and stable correlations have been identified.</p>   
      <p>Cluster determination using GAP statistics: Cross validation-based methods for cluster number estimation have been applied to estimate the number of stable clusters in each</p><p>dataset and cultivation year.</p>   
      <p>Regularized regression: Regularized regression has been used for feature selection and phenotype prediction, including ridge regression, elastic net and LASSO.</p>  
      <p>Software used in frame of the project: R, Excel, Compass DataAnalysis, QuantAnalysis and MetaboScape (Bruker, Bremen)</p>  
      <p>The work will be submitted for publication, hopefully within a few months.</p>
      <p></p><br>
      <p></p><br>
    <h5><p>Project: Integration of biotic and abiotic stress responses in barley</p></h5>
    <p style="font-size: 14px">June 2018 - March 2020</p>
        <p>The project aimed on understanding the coordination of the stress responses of barley. My task was to characterize cellular</p>
        <p>reactions of a diverse barley population to a biotic and abiotic stressor. As abiotic stress the plants were exposed to cold and light.</p>
        <p>Biotic stress was a pathogenic fungi.</p>
        <p>To evaluate the metabolic response of a variety of barley to different kinds of stress, semi-polar compounds were analyzed using UPLC-MS/MS.</p>
        <p>Enzymatic and non-enzymatic components of the oxidative stress response were analyzed. For non-invasive characterization, parameters of the</p>
        <p>photosynthesis were measured as fluorescence signals of the leaves.</p>
    <p></p><br>
    <p></p><br>
    <p><a style="color:blue;" href="#Jumpup">up</a></p>
  </div>
  
  <div class="phytochamber">
  </div>
  
  <div class="textSection">
    <h3><a id="Jump2" style="text-decoration: none">Master thesis</a></h3>
    <h5>at the Fraunhofer Institute for Molecular Biology and Applied Ecology IME Aachen</h5>
    <h5>Title: Exploration of pseudogenes related to disease resistance in potato</h5>
      <p>Abstract Pseudogenes have mostly been considered to be useless leftovers of once functional genes. However, a new view emerged when pseudogenes were</p>
      <p>discovered to have beneficial properties. One aspect is the perception of pseudogenes acting as a storage for genetic diversity from which novel functions</p>
      <p>can arise. This observation led to the notion of resistance gene-like pseudogenes as a source for new resistances. Pseudogenes are non-functional due to premature</p>
      <p>stop codons or frameshift mutations. The pre-sent thesis is part of a research project, which seeks to identify these genetic degenerations and restore the full-length</p>
      <p>protein by alterations on the single nucleotide level. The present work aims to contribute to the identification of pseudogenes by validating the presence of selected</p>
      <p>pseudogenes and characterizing their regulatory regions. Potato was chosen for this approach due to its high importance as a food crop and great efforts in resistance</p>
      <p>breeding to reduce crop losses. We focused on two pseudogenes, which were validated by PCR amplification and sequencing. Subsequently, we located the putative promoter</p>
      <p>region and assessed its activity. The intent of the promoter identification was to use the native promoter to regulate the altered pseudogene after restoration. The</p>
      <p>putative promoter of one selected pseudogene was identified by PCR walking and sequencing. The region was inserted into a vec-tor upstream of the reporter gene DsRed.</p>
      <p>To test the promoter activity, Nicotiana benthamiana was transformed with the construct. We observed local expression of DsRed, indicating promoter activity. However, the</p>
      <p>findings were difficult to reproduce, so that at the current time, no certain statements can be made as to whether the observed promoter activity is inducible by specific</p>
      <p>conditions.</p>
    <p></p><br>
    <p></p><br>
    <p><a style="color:blue;" href="#Jumpup">up</a></p>
  </div>
  
  <div class="desk">
  </div>
  
  <div class="textSection">
    <h3><a id="Jump3" style="text-decoration: none">Master of Science, Biology</a></h3>
    <h5>at Rheinisch Westfälische Technische Hochschule RWTH</h5>
      <p>Content:</p>
      <p>- Cell Biology</p>
      <p>- Genetics of Plant-Microbe-Interaction</p>
      <p>- Molecular Plant Physiology</p>
      <p>- Molecular Virology<p>
      <p>- Plant Cell walls</p>
    <h3><a id="Jump4" style="text-decoration: none">Reserach Internship</a></h3>
    <h5>at the Institue for Plant Physiolgy</h5>
      <p>Content: Molecular Biology and Plant Physiology</p>
    
</html>
