<html>
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="style.css">
  </head>
  
  <div class="hplc">
    <div class="caption">
      <span class="border" >Sina Marquardt, M.Sc. Biology</span>
    </div>
    <div class="caption_subtitle">
      <span class="border_subtitle" >Biochemistry. Molecular Biology. Bioinformatics.</span>
    </div>
  </div>
  
  <div class="textSection">
    <h3><a id="Jumpup" style="text-decoration: none">Professional Experience</a></h3>
    
    <table style="width:850px">
      <tr>
        <td>
          <h4><a href="#Jump1">Scientific employee</a></h4>
          <p style="line-height: 0%">IPK Gatersleben</p>
          <p style="font-size: 14px">Juni 2018 - November 2020</p>
        </td>
      </tr>
      
      <tr>
        <td>
          <h4><a href="#Jump2">Master thesis</a></h4>
          <p style="line-height: 0%">Fraunhofer IME</p>
          <p style="font-size: 14px">April 2017 - Dezember 2017</p>
        </td>
      </tr>
                 
      <tr>
        <td>
          <h4>Research Internship</h4>
          <p style="line-height: 0%">RWTH University Aachen</p>
          <p style="font-size: 14px">Juli 2016 - Septeber 2016</p>
        </td>
      </tr>
      
      <tr>
        <td>
          <h4>Master of Science, Biology</h4>
          <p style="line-height: 0%">RWTH Aachen University</p>
          <p style="font-size: 14px">October 2012 - January 2018</p>
        </td>
      </tr>
      
      <tr>
        <td>
          <h4>Bachelor thesis</h4>
          <p style="line-height: 0%">RWTH Aachen University Hospital</p>
          <p style="font-size: 14px">April 2015 - September 2015</p>
        </td>
      </tr>

      <tr>
        <td>
          <h4>Bachelor of Science, Biology</h4>
          <p style="line-height: 0%">RWTH Aachen University</p>
          <p style="font-size: 14px">October 2012 - September 2015</p>
        </td>
      </tr>
    
    </table>
    
    <p></p><br>
    <h3>Technical Skills</h3>
    <table style="width:850px">
      <tr>
        <td>
            <h5>Biological Methods:</h5>
              <p>- Molekularbiologische Methoden (PCR, Genome Walking, Klonierung, Präparation von genomischer und Plasmid-DNA, DNA und Protein Gel Elektrophorese, Immunoblotting)</p>
              <p>- Fluoreszenz- und Massenspektrometrie (LC-MS, MALDI-TOF-MS)</p>
              <p>- Kultivierung und Transformation von Mikroorganismen und Pflanzen</p> 
        </td>
      </tr>
      
      <tr>
        <td>      
            <h5>Software and Programming Skills:</h5>
              <p>- Sequenzanalyse (CloneManager)</p>
              <p>- Statistische Datenanalyse (R, SPSS)</p>
              <p>- Laborinformationsmanagement (LIMSOPHY)</p>
              <p>- Microsoft Office (Word, Excel, PowerPoint)</p>
              <p>- Massenspektrometrische Analyse (Bruker Software: DataAnalysis, Metaboscape, QuantAnalysis)</p>
              <p>- Graphische Bearbeitung (inkscape, GIMP)</p>
              <p>- Programming (Python and html basics)</p>
        </td>
      </tr>
    </table>

    <h3><p></p><br><p>Contact</p></h3>
    
    <p>
      <a href="http://linkedin.com/in/sina-marquardt-8b4225156" style="text-decoration: underline" class="author-social" target="_blank">linkedin</a>
      <a href="https://github.com/sina-marquardt" style="text-decoration: underline" target="_blank">github</a>
    </p>
  </div>
  
  <div class="hplc_room">
  </div>
  
  <div class="textSection">
    <h3><a id="Jump1" style="text-decoration: none">Scientific employee</a></h3>
    <h5>at the Leibniz Institute for Plant Genetics and Crop Plant Research IPK Gatersleben</h5>
    <h5>Department of Physiology and Cell Biology / Applied Biochemistry</h5>
    <h5><p>Project: BARLEY BIODIVERSITY - Increasing biodiversity and productivity of barley cultivars using a sensor-based high-throughput phenotyping system in the field and association mapping with two wild barley introgression libraries</p></h5>
    <p style="font-size: 14px">Juni 2018 - November 2020</p>
    <p>The project was part of the initiative „Innovative Pflanzenzüchtung im Anbausystem“ (IPAS)</p>
    <p>Tasks and methods: My task in the project was the evaluation and statistical analysis of multi-modal data from a large barley maping population.The data have been obtained</p><p>using various analytical techniques and across several years. In terms of analytical complexity, the project included handling large and diverse data structures, including:</p><p>genotype data, mass spectrometry profiles, phenotype descriptors and quantitative phenotypes, as well as analysis of the data variation and reproducibility across different</p><p>cultivation years and plant developmental stages.</p> 
Analysis of metabolic data:Identification of metabolic features in the mass spectrometry data obtained by HPLC-MS/MS, providing a comprehensive profile of semi-polar compounds accumulated in barley leaves
Data normalization:Data normalization using multiple internal standards by fitting a multivariate linear model to a set of reference internal standards and removing the fitted variance from the target variables.
Quality control: Quality control (QC) samples were used to estimate the data reproducibility and technical error across analytical batches and cultivation seasons. 
ANOVA: Analysis of variance has been performed to highlight effects of different experimental factors: e.g. genotype, batch, developmental stage. 
Pearson correlation analysis: Correlation analysis has been performed between measured variables using Pearson’s correlation coefficient. Homogeneity of the correlations across cultivation seasons has been analysed and stable correlations have been identified. 
Cluster determination using GAP statistics: Cross validation-based methods for cluster number estimation have been applied to estimate the number of stable clusters in each dataset and cultivation year. 
Regularized regression: Regularized regression has been used for feature selection and phenotype prediction, including ridge regression, elastic net and LASSO.
Software used in frame of the project: R, Excel, Compass DataAnalysis, QuantAnalysis and MetaboScape (Bruker, Bremen)
The work will be submitted for publication, hopefully within a few months</p>
    <p></p><br>
    <p></p><br>
    <p><a style="color:blue;" href="#Jumpup">up</a></p>
  </div>
  
  <div class="phytochamber">
  </div>
  
  <div class="textSection">
    <h3><a id="Jump2" style="text-decoration: none">Master thesis</a></h3>
    <h5>at the Fraunhofer Institute for Molecular Biology and Applied Ecology IME Aachen</h5>
    <h5>Title: Exploration of pseudogenes related to disease resistance in potato</h5>
    <p>text text text</p>
    <p>text text text</p>
    <p>text text text</p>
    <p></p><br>
    <p></p><br>
    <p><a style="color:blue;" href="#Jumpup">up</a></p>
  </div>
  
</html>
